composer.lock

```diff
@@ -18116,7 +18167,7 @@
         },
         {
             "name": "drupal/core-dev",
-            "version": "9.3.12",
+            "version": "9.3.9",
```

Comment:
We don't want this downgrade. I think that this happenned via an auto-merge when I clicked the 'Update branch' button. Please reroll, despite the PR saying there are no conflicts.

Reply:
@weitzman just updated. Thanks.